### PR TITLE
Fix BA124 case sensitive

### PR DESCRIPTION
**Pull request by anas-yousef**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11278

### CODE REVIEW

File: demisto_sdk/commands/validate/tests/BA_validators_test.py [hunk 1]
```diff
@@ -2068,3 +2077,58 @@ def test_IsFolderNameHasSeparatorsValidator_obtain_invalid_content_items(
...
+)
+def test_IsHaveUnitTestFileValidator_is_valid(
+    content_items, expected_number_of_failures, expected_msgs
+):
+    """
+    Given
+    content_items list.
+        - Case 1: Three content items, where the last one has an invalid unit test file name.
+    When
+    - Calling the IsHaveUnitTestFileValidator is_valid function.
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/tests/BA_validators_test.py:2119]:
```suggestion
def test_IsHaveUnitTestFileValidator_obtain_invalid_content_items(
    content_items, expected_number_of_failures, expected_msgs
):
    """
    Given
    content_items list.
        - Case 1: Three content items, where the last one has an invalid unit test file name.
    When
    - Calling the IsHaveUnitTestFileValidator obtain_invalid_content_items function.
```

File: demisto_sdk/commands/validate/tests/BA_validators_test.py [hunk 2]
```diff
@@ -2068,3 +2077,58 @@ def test_IsFolderNameHasSeparatorsValidator_obtain_invalid_content_items(
...
+    Then
+        - Make sure the right amount of failures return and that the error msg is correct.
+    """
+    create_indicator_type_object()
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/tests/BA_validators_test.py:2123]:
```suggestion
```
?

**Review by YuvHayun — APPROVED** [review]
Nice work, minor comments.

File: demisto_sdk/commands/validate/tests/BA_validators_test.py (hunk 2 above)
Comment (anas-yousef) [reply demisto_sdk/commands/validate/tests/BA_validators_test.py:2123]: By accident, forgot to remove it

**Pull request merged**